ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/route_maps.py

```diff
@@ -133,26 +125,35 @@ def route_maps(self) -> list | None:
             # RM-BGP-UNDERLAY-PEERS-OUT
             sequence_numbers = [
                 {
-                    "sequence": 10,
+                    "sequence": 20,
                     "type": "permit",
                     "description": "Advertise local routes towards LAN",
-                    "match": ["extcommunity ECL-EVPN-SOO"],
+                    "match": ["route-type local"],
                 },
                 {
-                    "sequence": 20,
+                    "sequence": 30,
                     "type": "permit",
```

Comment:
These are breaking changes. Can we revert and submit an issue for 5.0 to adjust this?

Reply:
yeah so these ... are breaking changes - should we do differently for HA / non HA ?

Reply:
introduced an if/else with a TODO